core/templates/pages/exploration-editor-page/translation-tab/audio-translation-bar/audio-translation-bar.directive.html

```diff
@@ -152,10 +152,10 @@
     </div>
   </div>
   <div class="oppia-content-wrapper oppia-recording-timer">
-    <div ng-if="isAudioAvailable && !isPlayingUploadedAudio()">
+    <div ng-if="isAudioAvailable && isLoadingAudio">
       <[0 | formatTimer]> / <[durationSecs | formatTimer]>
     </div>
-    <div ng-if="audioTimerIsShown && isPlayingUploadedAudio()">
+    <div ng-if="audioTimerIsShown">
```

Comment:
Just checking, should this show when the audio is loading?

Reply:
Line 158? no that is only shown when the audio is downloaded completely.

Reply:
Oh, in that case do we need a `!AudioIsLoading` condition here?

Reply:
For readability's sake, I think will add it there.

Reply:
done